### PR TITLE
DOCSP-5379: Pull assets from docs-tools

### PR DESCRIPTION
**Pull request by i80and**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,16 +30,31 @@ deploy: build
...
+		cp ${DOCS_TOOLS_THEME}/navbar.min.js* $$prefix/js || exit 1; \
+		cp ${DOCS_TOOLS_THEME}/landing.css $$prefix/css || exit 1; \
+		cp ${DOCS_TOOLS_THEME}/navbar.min.css $$prefix/css || exit 1; \
+	done
+
 	@# Run the script to generate each landing page
 	python3 ./gen_landings.py $@
+
+clean:
+	-rm -r build
```
Comment (schmalliso) [inline Makefile]: [for my edification] what does the leading `-` do here?

File: Makefile [hunk 2]
```diff
@@ -31,16 +30,31 @@ deploy: build
...
 	@# Copy CSS and JS files to output directories
 	cp static/favicon.png $@/favicon.ico
-	cp -r static/images static/css static/js $@/
-	cp -r static/images static/css static/js $@/tools
-	cp -r static/images static/css static/js $@/cloud
+	for prefix in $@/ $@/tools $@/cloud; do \
+		mkdir -p $$prefix/js || exit 1; \
+		cp -r static/images $$prefix || exit 1; \
+		cp -r static/css $$prefix || exit 1; \
+		cp ${DOCS_TOOLS_THEME}/landing.min.js* $$prefix/js || exit 1; \
```
Comment (schmalliso) [inline Makefile]: Getting an error on build because`docs-tools` folder is empty - seems like the submodule isn't loading?

Comment (schmalliso) [reply Makefile]: Okay, I've run `git submodule init` and now docs-tools has stuff in it - is that something we need to do when we merge this sucker, or should everyone who builds docs-landing run `git submodule init`, or is it a Makefile thing we should include or… something else?

File: Makefile [hunk 3]
```diff
@@ -31,16 +30,31 @@ deploy: build
...
+
 	@# Copy CSS and JS files to output directories
 	cp static/favicon.png $@/favicon.ico
-	cp -r static/images static/css static/js $@/
-	cp -r static/images static/css static/js $@/tools
-	cp -r static/images static/css static/js $@/cloud
+	for prefix in $@/ $@/tools $@/cloud; do \
+		mkdir -p $$prefix/js || exit 1; \
+		cp -r static/images $$prefix || exit 1; \
+		cp -r static/css $$prefix || exit 1; \
```
Comment (schmalliso) [inline Makefile]: I had a comment about the feedback widget CSS file extension-ing being crazy, but Sophie has fixed that in docs-tools so it's not a problem here anymore! Yay!

File: Makefile (hunk 1 above)
Comment (i80and) [reply Makefile]: If the command fails (e.g. if `build` doesn't exist), the rule isn't considered to have failed, and Make won't print an error message

**Comment by schmalliso** [comment]
@i80and I've rebased this sucker and tested and it works a charm, can we just merge it so we don't have to futz around for the next World Build?

**Pull request merged**